crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -154,9 +162,9 @@ impl SlidingSync {
     }
 
     /// Unsubscribe from a given room.
-    pub fn unsubscribe(&self, room_id: OwnedRoomId) -> Result<()> {
-        if self.inner.subscriptions.write().unwrap().remove(&room_id).is_some() {
-            self.inner.unsubscribe.write().unwrap().push(room_id);
+    pub fn unsubscribe_to_room(&self, room_id: OwnedRoomId) -> Result<()> {
```

Comment:
"unsubscribe to" is wrong grammatically. It should be "unsubscribe from".

Reply:
Haha, yeah, indeed! Thanks!